recipe/meta.yaml

```diff
@@ -25,35 +25,34 @@ build:
 requirements:
   host:
     - python
-    - debugpy >=1.0.0
-    - hatchling >=1.4
-    - ipython >=7.23.1
-    - jupyter_client >=6.1.12
-    - packaging
     - pip
-    - comm >=0.1
-    - psutil
+    - setuptools
     - wheel
```

Comment:
`wheel` can be removed when `hatchling` is used, and I think `setuptools` can also be removed.